licenses-binary/LICENSE-aopalliance.txt

```diff
@@ -0,0 +1,4 @@
+Text copied verbatim from http://aopalliance.sourceforge.net/
```

Comment:
@felixcheung Is this good enough for `public domain` license?

Reply:
https://apache.org/legal/resolved.html#handling-public-domain-licensed-works

do you see a reference to a license?

Reply:
There is just that one line there in the page http://aopalliance.sourceforge.net/, with no link to the license clauses. There is a list of members right below the line that mentions LICENSE

Reply:
@mcvsubbu I think that we can remove this line and just keep the following line to this file.

```
LICENCE: all the source code provided by AOP Alliance is Public Domain.
```